controllers/homeRoutes.js

```diff
@@ -0,0 +1,57 @@
+const sequelize = require("../config/connection");
+const { Department, User, Product } = require("../models");
+const router = require("express").Router();
+
+// gather homepage route for all departments
+router.get("/", async (req, res) => {
+  try {
+    const departmentData = await Department.findAll({});
+
+    const departments = departmentData.map((Department) =>
+      Department.get({ plain: true })
+    );
+
+    console.log(departments);
+
+    // will set this to render homepage later on, for now just testing
+    return res.json(departments);
+  } catch (err) {
+    res.status(500).json(err);
+  }
+});
+
+// homepage route for a single department
+router.get("/departments/:id", async (req, res) => {
+  try {
+    const departmentData = await Department.findOne({
+      where: {
+        id: req.params.id,
+      },
+    });
+
+    const departments = departmentData.get({ plain: true });
+
+    console.log(departments);
+
+    // will set this to render homepage later on, for now just testing
+    return res.json(departments);
+  } catch (err) {
+    res.status(500).json(err);
```

Comment:
For the return statements we'd probably want to include a status code right? Or does it return a 200 automatically? I've been doing it like:

`return res.status(200).json(departments)`